packages/ui-extensions/src/surfaces/customer-account/api/order-status/order-status.ts

```diff
@@ -344,7 +148,7 @@ export interface Market {
   handle: string;
 }
 
-export interface Localization {
+export interface OrderStatusLocalization {
```

Comment:
Reason: `Localization` exists on both OrderStatusApi and StandardApi. Rename it to resolve the conflict. The same reason for the changes follows.